### PR TITLE
chore(css): Move CSS examples - Feature queries, overflow, subgrid, masonry

### PR DESCRIPTION
**Pull request by bsmth**
### Description

As part of an initiative to reduce repo maintenance & code duplication, this PR moves some external examples into content. This PR converts the following macros to live samples:

```
{{EmbedGHLiveSample("css-examples/feature-queries/simple.html", '100%', 600)}}
{{EmbedGHLiveSample("css-examples/feature-queries/not.html", '100%', 600)}}
{{EmbedGHLiveSample("css-examples/feature-queries/and.html", '100%', 600)}}
{{EmbedGHLiveSample("css-examples/feature-queries/or.html", '100%', 600)}}
{{EmbedGHLiveSample("css-examples/flow/overflow/overflow.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flow/overflow/hidden.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flow/overflow/scroll.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flow/overflow/auto.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flow/overflow/overflow-y.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flow/overflow/text-overflow.html", '100%', 500)}}
{{EmbedGHLiveSample("css-examples/flow/writing-modes/creative-use.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/flow/writing-modes/horizontal-tb.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/flow/writing-modes/vertical-rl.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/flow/writing-modes/vertical-lr.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/flow/writing-modes/inline-change-mode.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/flow/writing-modes/block-change-mode.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/flow/writing-modes/replaced.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/flow/writing-modes/width.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/flow/writing-modes/inline-size.html", '100%', 720)}}
{{EmbedGHLiveSample("css-examples/flow/in-flow/in-flow.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/flow/in-flow/float.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/flow/in-flow/abspos.html", '100%', 700)}}
{{EmbedGHLiveSample("css-examples/flow/in-flow/relative.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/grid/masonry/block-axis.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/grid/masonry/inline-axis.html", '100%', 1000)}}
{{EmbedGHLiveSample("css-examples/grid/masonry/spanners.html", '100%', 800)}}
{{EmbedGHLiveSample("css-examples/grid/masonry/positioned.html", '100%', 1000)}}
{{EmbedGHLiveSample("css-examples/grid/subgrid/columns.html", '100%', 1200)}}
{{EmbedGHLiveSample("css-examples/grid/subgrid/rows.html", '100%', 1200)}}
{{EmbedGHLiveSample("css-examples/grid/subgrid/both.html", '100%', 1200)}}
{{EmbedGHLiveSample("css-examples/grid/subgrid/no-implicit.html", '100%', 1200)}}
{{EmbedGHLiveSample("css-examples/grid/subgrid/implicit.html", '100%', 1200)}}
{{EmbedGHLiveSample("css-examples/grid/subgrid/gap.html", '100%', 1200)}}
{{EmbedGHLiveSample("css-examples/grid/subgrid/line-names.html", '100%', 1200)}}
{{EmbedGHLiveSample("css-examples/grid/subgrid/adding-line-names.html", '100%', 1200)}}
```

### Motivation

Motivation is described in the project issue.

### Related issues and pull requests

- [x] https://github.com/mdn/mdn/issues/597

### CODE REVIEW

File: files/en-us/web/css/css_conditional_rules/using_feature_queries/index.md [hunk 1]
```diff
@@ -40,7 +40,31 @@ For example, we can apply a set of styles or import an entire stylesheet if the
 
 As another example, if you want to check if a browser supports the `row-gap` property you would write the following feature query. It doesn't matter which value you use in a lot of cases: if all you want is to check that the browser supports this property, then any valid value will do.
 
-{{EmbedGHLiveSample("css-examples/feature-queries/simple.html", '100%', 600)}}
+```html live-sample___simple
+<div class="box">
+  If your browser supports the row-gap property, the border will be dashed and
+  text will be red
```
Comment (estelle) [inline files/en-us/web/css/css_conditional_rules/using_feature_queries/index.md:46]:
```suggestion
  text will be red.
```

File: files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md [hunk 2]
```diff
@@ -7,59 +7,268 @@ page-type: guide
...
+```html live-sample___overflow
+<div class="box">
+  <p>
+    One November night in the year 1782, so the story runs, two brothers sat
+    over their winter fire in the little French town of Annonay, watching the
+    grey smoke-wreaths from the hearth curl up the wide chimney.
+  </p>
+</div>
+<p>
+  Their names were Stephen and Joseph Montgolfier, they were papermakers by
```
Comment (estelle) [inline files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md:25]:
```suggestion
  Their names were Stephen and Joseph Montgolfier. They were papermakers by
```

File: files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md [hunk 3]
```diff
@@ -7,59 +7,268 @@ page-type: guide
...
+```html hidden live-sample___hidden
+<div class="box">
+  <p>
+    One November night in the year 1782, so the story runs, two brothers sat
+    over their winter fire in the little French town of Annonay, watching the
+    grey smoke-wreaths from the hearth curl up the wide chimney.
+  </p>
+</div>
+<p>
+  Their names were Stephen and Joseph Montgolfier, they were papermakers by
```
Comment (estelle) [inline files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md:66]:
```suggestion
  Their names were Stephen and Joseph Montgolfier. They were papermakers by
```

File: files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md [hunk 4]
```diff
@@ -7,59 +7,268 @@ page-type: guide
...
+  trade, and were noted as possessing thoughtful minds and a deep interest in
+  all scientific knowledge and new discovery.
```
Comment (estelle) [inline files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md:105]:
```suggestion
Their names were Stephen and Joseph Montgolfier; they were papermakers by trade and were noted as possessing thoughtful minds and a deep interest in all scientific knowledge and new discoveries.
```
per Grammarly ;)

File: files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md [hunk 5]
```diff
@@ -7,59 +7,268 @@ page-type: guide
...
+## Flow Relative Properties
 
-These properties currently do not have implementations in browsers, so you will need to use the physical properties at the present time and adjust for your writing mode.
+In the guide to [Writing Modes and Flow Layout](/en-US/docs/Web/CSS/CSS_flow_layout/Flow_layout_and_writing_modes), we looked at the `block-size` and `inline-size` properties, which make more sense when working with different writing modes than tying our layout to the physical dimensions of the screen. The Level 3 Overflow Module also includes flow relative properties for overflow - [`overflow-block`](/en-US/docs/Web/CSS/@media/overflow-block) and [`overflow-inline`](/en-US/docs/Web/CSS/@media/overflow-inline). These correspond to `overflow-x` and `overflow-y` but the mapping depends on the writing mode of the document.
```
Comment (estelle) [inline files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md:210]:
```suggestion
In the guide to [Writing Modes and Flow Layout](/en-US/docs/Web/CSS/CSS_flow_layout/Flow_layout_and_writing_modes), we looked at the `block-size` and `inline-size` properties, which make more sense when working with different writing modes than tying our layout to the physical dimensions of the screen. The [CSS overflow module](/en-US/docs/Web/CSS/CSS_overflow) also includes flow relative properties for overflow - [`overflow-block`](/en-US/docs/Web/CSS/@media/overflow-block) and [`overflow-inline`](/en-US/docs/Web/CSS/@media/overflow-inline). These correspond to `overflow-x` and `overflow-y` but the mapping depends on the writing mode of the document.
```

File: files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md [hunk 6]
```diff
@@ -7,59 +7,268 @@ page-type: guide
...
 
 ## Indicating Overflow
 
-In the Level 3 Overflow specification we have some properties which can help improve the way content looks in an overflow situation.
+In the Level 3 Overflow specification, there are some properties which can help improve the way content looks in an overflow situation.
```
Comment (estelle) [inline files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md:214]:
```suggestion
In the CSS overflow module, there are some properties that can help improve the way content looks in an overflow situation.
```

File: files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md [hunk 7]
```diff
@@ -7,59 +7,268 @@ page-type: guide
...
 
 ## Summary
 
 Whether you are in continuous media on the web or in a Paged Media format such as print or EPUB, understanding how content overflows is useful when dealing with any layout method. By understanding how overflow works in normal flow, you should find it easier to understand the implications of overflow content in layout methods such as grid and flexbox.
+
+## See also
+
+- [Overflowing content](/en-US/docs/Learn/CSS/Building_blocks/Overflowing_content) guide
+- [CSS overflow](/en-US/docs/Web/CSS/CSS_overflow)
+- [CSS containment](/en-US/docs/Web/CSS/CSS_containment)
```
Comment (estelle) [inline files/en-us/web/css/css_flow_layout/flow_layout_and_overflow/index.md:274]:
```suggestion
- [CSS overflow](/en-US/docs/Web/CSS/CSS_overflow) module
- [CSS containment](/en-US/docs/Web/CSS/CSS_containment) module
```

File: files/en-us/web/css/css_flow_layout/in_flow_and_out_of_flow/index.md [hunk 8]
```diff
@@ -8,11 +8,51 @@ page-type: guide
 
 The [previous guide](/en-US/docs/Web/CSS/CSS_flow_layout/Block_and_inline_layout_in_normal_flow) explained block and inline layout in normal flow. All elements that are in flow will be laid out using this method.
 
-The following example contains a heading, paragraph, a list and a final paragraph which contains a `strong` element. The heading and paragraphs are block level, the `strong` element inline. The list is displayed using flexbox to arrange the items into a row, however it too is participating in block and inline layout - the container has an outside `display` type of `block`.
+## Example of elements in flow
 
-{{EmbedGHLiveSample("css-examples/flow/in-flow/in-flow.html", '100%', 800)}}
+The following example contains a heading, paragraph, a list and a final paragraph which contains a `strong` element. The heading and paragraphs are block level, the `strong` element inline. The list is displayed using flexbox to arrange the items into a row, however it too is participating in block and inline layout - the container has an outside `display` type of `block`.
```
Comment (estelle) [inline files/en-us/web/css/css_flow_layout/in_flow_and_out_of_flow/index.md:13]:
```suggestion
The following example contains a heading, a paragraph, a list, and a final paragraph that contains a `strong` element. The heading and paragraphs are block level, the `strong` element inline. The list, which is displayed using flexbox to arrange the items into a row, is also participating in block and inline layout - the container has an outside `display` type of `block`.
```

File: files/en-us/web/css/css_flow_layout/in_flow_and_out_of_flow/index.md [hunk 9]
```diff
@@ -8,11 +8,51 @@ page-type: guide
...
+  display: flex;
+  justify-content: space-around;
+  list-style: none;
+  margin: 0;
+}
+```
+
+{{EmbedLiveSample("in-flow", "", "300px")}}
+
+All of the elements can be said to be in flow, appearing on the page in the order that they are in the source.
```
Comment (estelle) [inline files/en-us/web/css/css_flow_layout/in_flow_and_out_of_flow/index.md:55]:
```suggestion
All of the elements are considered "in flow"; they appear on the page in the order that they are in the source.
```

File: files/en-us/web/css/css_flow_layout/in_flow_and_out_of_flow/index.md [hunk 10]
```diff
@@ -22,23 +62,100 @@ All elements are in-flow apart from:
 - items with `position: absolute` (including `position: fixed` which acts in the same way)
 - the root element (`html`)
 
-Out of flow items create a new Block Formatting Context (BFC) and therefore everything inside them can be seen as a mini layout, separate from the rest of the page. The root element therefore is out of flow, as the container for everything in our document, and establishes the Block Formatting Context for the document.
+Out of flow items create a new [Block Formatting Context](/en-US/docs/Web/CSS/CSS_display/Block_formatting_context) (BFC) and therefore everything inside them can be seen as a mini layout, separate from the rest of the page. The root element therefore is out of flow, as the container for everything in our document, and establishes the Block Formatting Context for the document.
```
Comment (estelle) [inline files/en-us/web/css/css_flow_layout/in_flow_and_out_of_flow/index.md:65]:
```suggestion
Out-of-flow items create a new [Block Formatting Context](/en-US/docs/Web/CSS/CSS_display/Block_formatting_context) (BFC), and therefore, everything inside them can be seen as a mini layout, separate from the rest of the page. The root element, therefore, is out of flow, as the container for everything in our document, and establishes the Block Formatting Context for the document.
```

File: files/en-us/web/css/css_grid_layout/masonry_layout/index.md [hunk 11]
```diff
@@ -17,24 +17,105 @@ Masonry layout is a layout method where one axis uses a typical strict grid layo
 
 ## Creating a masonry layout
 
-To create the most common masonry layout, your columns will be the grid axis and the rows the masonry axis. Define this layout with `grid-template-columns` and `grid-template-rows`:
+To create the most common masonry layout, your columns will be the grid axis and the rows the masonry axis. Define this layout with `grid-template-columns` and `grid-template-rows`.
```
Comment (estelle) [inline files/en-us/web/css/css_grid_layout/masonry_layout/index.md:20]:
```suggestion
To create the most common masonry layout, your columns will be the grid axis and the rows the masonry axis, defined with `grid-template-columns` and `grid-template-rows`.
```

File: files/en-us/web/css/css_grid_layout/masonry_layout/index.md [hunk 12]
```diff
@@ -17,24 +17,105 @@ Masonry layout is a layout method where one axis uses a typical strict grid layo
...
+The child elements of this container will now lay out item by item along the rows, as they would with regular grid layout automatic placement.
 
-```css
-.container {
+As they move onto a new row, the items will display according to the masonry algorithm. Items will load into the column with the most room causing a tightly packed layout without strict row tracks.
```
Comment (estelle) [inline files/en-us/web/css/css_grid_layout/masonry_layout/index.md:23]:
```suggestion
As the items move onto new rows, they will display according to the masonry algorithm. Items will load into the column with the most room, causing a tightly packed layout without strict row tracks.
```

File: files/en-us/web/css/css_grid_layout/masonry_layout/index.md [hunk 13]
```diff
@@ -17,24 +17,105 @@ Masonry layout is a layout method where one axis uses a typical strict grid layo
...
+  <div class="item" style="block-size: 3.5em;"></div>
+  <div class="item" style="block-size: 2.8em;"></div>
+</div>
+```
 
-{{EmbedGHLiveSample("css-examples/grid/masonry/block-axis.html", '100%', 800)}}
+{{EmbedLiveSample("block-axis", "", "250px")}}
 
 It is also possible to create a masonry layout with items loading into rows.
+Identical HTML is used in the following example, so it's omitted for brevity:
```
Comment (estelle) [inline files/en-us/web/css/css_grid_layout/masonry_layout/index.md:72]:
```suggestion
```
the user doesn't know it's a different example (it's under the same header), so let's remove as more confusing than clarifying.

File: files/en-us/web/css/css_flow_layout/in_flow_and_out_of_flow/index.md [hunk 14]
```diff
@@ -48,7 +165,48 @@ When taking an item out of flow with positioning, you will need to manage the po
...
+  </p>
+  <p class="relative">
+    Their names were Stephen and Joseph Montgolfier, they were papermakers by
+    trade, and were noted as possessing thoughtful minds and a deep interest in
+    all scientific knowledge and new discovery.
+  </p>
+  <p>
+    Before that night—a memorable night, as it was to prove—hundreds of millions
+    of people had watched the rising smoke-wreaths of their fires without
+    drawing any special inspiration from the fact.”
```
Comment (estelle) [inline files/en-us/web/css/css_flow_layout/in_flow_and_out_of_flow/index.md:183]:
```suggestion
    drawing any special inspiration from the fact.
```

**Review by estelle — COMMENTED** [review]
There are a few issues with the text that is used in all the examples. Some edits are optional, but others do need to be fixed (even though it's likely like with trailing`"`.

the examples do all work, so that is approvable, but let's remove the "HTML hidden for brevity" and any other nits you agree with me on.

File: files/en-us/web/css/css_flow_layout/in_flow_and_out_of_flow/index.md (hunk 14 above)
Comment (estelle) [reply files/en-us/web/css/css_flow_layout/in_flow_and_out_of_flow/index.md:183]: 3 times.

Comment (bsmth) [reply files/en-us/web/css/css_flow_layout/in_flow_and_out_of_flow/index.md]: fixed in cad2c985d5

File: files/en-us/web/css/css_grid_layout/masonry_layout/index.md (hunk 13 above)
Comment (bsmth) [reply files/en-us/web/css/css_grid_layout/masonry_layout/index.md]: fixed in cad2c985d5

**Comment by bsmth** [comment]
Thanks, Estelle! Should be ready for another look 👀

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Pull request merged**